Makefile

```diff
@@ -0,0 +1,38 @@
+
+DOCKER := $(shell command -v docker 2> /dev/null)
+DOCKER_COMPOSE := $(shell command -v docker-compose 2> /dev/null)
+
+dchecks:
+ifndef DOCKER
+	$(error "Command is not available please install Docker")
+endif
+ifndef DOCKER_COMPOSE
+	$(error "Command is not available please install docker-compose")
+endif
+
+setup: checks ## Docker: setup, build/rebuild app container
+	cp --no-clobber sample.env .env
```

Comment:
`sample.env` file is neither in the repo nor in this commit. Actually, `.gitignore` file has an entry `*.env` (which should be `.env`) that might be preventing it from being tracked.